app/robots/iCubGenova02/hardware/motorControl/left_upper_leg-ems6-mc.xml.26apr16

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Oh yes will remove the hardware directory. It is not needed
On Aug 29, 2016 16:55, "Silvio Traversaro" <notifications@github.com> wrote:

> In app/robots/iCubGenova02/hardware/motorControl/left_
> upper_leg-ems6-mc.xml.26apr16
> <https://github.com/robotology/icub-main/pull/363#discussion_r76621355>:
>
> > @@ -0,0 +1,66 @@
> > +<?xml version="1.0" encoding="UTF-8" ?>
>
> See https://github.com/robotology/icub-main/pull/363/files#r76621304 .
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/robotology/icub-main/pull/363/files/8ce0a4b1e1a036910224caa1ec01c0b49cd26727#r76621355>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGNGDtApNzDXSTyes7Ge0NahDQD2U3hgks5qkvLggaJpZM4JvjcF>
> .
>